kroxylicious-filter-test-support/src/main/java/io/kroxylicious/test/record/RecordTestUtils.java

```diff
@@ -85,6 +86,19 @@ public static Record record(ByteBuffer value,
         return record(RecordBatch.CURRENT_MAGIC_VALUE, 0, 0, null, value, headers);
     }
 
+    /**
+     * Return a Record with the given value, offset and headers
+     * @param value
+     * @param offset
+     * @param headers
+     * @return The record
+     */
+    public static Record record(ByteBuffer value,
+                                long offset,
+                                Header... headers) {
+        return record(RecordBatch.CURRENT_MAGIC_VALUE, offset, 0, null, value, headers);
```

Comment:
We should probably factor out constants for `DEFAULT_TIMESTAMP`, `DEFAULT_OFFSET` etc.

Reply:
introduced constants for the default record properties